### PR TITLE
[addon] fix wrong way to set addon version and level

### PR DESCRIPTION
**Pull request by AlwinEsch**
This fix a from me wrong inserted way to set Library Version and Level inside Kodi.

Also fixes this two libraries where it was not set before on right way (InputStream and Peripheral)

### CODE REVIEW

**Comment by AlwinEsch** [comment]
jenkins build this please

**Comment by AlwinEsch** [comment]
jenkins build this please

File: xbmc/addons/binary/interfaces/api1/Peripheral/AddonCallbacksPeripheral.cpp [hunk 1]
```diff
@@ -37,7 +37,7 @@ namespace Peripheral
 {
 
 CAddonCallbacksPeripheral::CAddonCallbacksPeripheral(ADDON::CAddon* addon)
-  : ADDON::IAddonInterface(addon, APILevel(), Version()),
+  : ADDON::IAddonInterface(addon, 1, "0.0.1"),
```
Comment (garbear) [inline xbmc/addons/binary/interfaces/api1/Peripheral/AddonCallbacksPeripheral.cpp]: maybe use `PERIPHERAL_API_VERSION`?

Comment (AlwinEsch) [inline xbmc/addons/binary/interfaces/api1/Peripheral/AddonCallbacksPeripheral.cpp]: Thanks, has not looked to the kodi_peripheral_types.h only the libKODI_peripheral.h

Becomes changed.

**Comment by AlwinEsch** [comment]
jenkins build this please again and please win against windows you are Super Jenkins :D

**Pull request merged**